week1/extra-homework/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
No need for this `package.json` file, since we haven't made a have a library and it's not requires any dependencies.

Reply:
I made this `package.json` just for keeping the extra-homework as a different app with app name and description. You're right that i didn't add any dependency to this app.